.drone.yml

```diff
@@ -50,18 +57,94 @@ steps:
       email: lucas@loeffel.io
       folder: npm
       access: public
-      # tag: beta
+      tag: beta
     when:
       event: tag
 
+---
+kind: pipeline
+type: docker
+name: docker-linux-amd64
+depends_on:
+  - main
+trigger:
+  event:
+    - tag
+
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+  - name: build
+    image: golang
+    commands:
+      - make build
   - name: docker hub
-    image: plugins/docker
+    image: plugins/docker:linux-amd64
     settings:
       username: loeffel
       password:
         from_secret: DOCKER_PASSWORD
       dockerfile: docker/Dockerfile
       auto_tag: true
+      auto_tag_suffix: linux-amd64
       repo: lslintorg/ls-lint
-    when:
-      event: tag
+
+---
+kind: pipeline
+type: docker
+name: docker-linux-arm64
+depends_on:
+  - main
+trigger:
+  event:
+    - tag
+
+platform:
+  os: linux
+  arch: arm64
+
+steps:
+  - name: build
+    image: golang
+    commands:
+      - make build
+  - name: docker hub
+    image: plugins/docker:linux-arm64
+    settings:
+      username: loeffel
+      password:
+        from_secret: DOCKER_PASSWORD
+      dockerfile: docker/Dockerfile.arm64
+      auto_tag: true
+      auto_tag_suffix: linux-arm64
+      repo: lslintorg/ls-lint
+
+---
+kind: pipeline
+type: docker
+name: docker-manifest
+depends_on:
+  - docker-linux-amd64
+  - docker-linux-arm64
+trigger:
+  event:
+    - tag
+
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+  - name: manifest
+    pull: always
+    image: plugins/manifest
+    settings:
+      auto_tag: true
+      ignore_missing: true
```

Comment:
why do we want to ignore missing source images?

Reply:
I don't think is needed. I just copied this from some other project.

Reply:
i removed it for now